project-backups.rst

```diff
@@ -59,9 +60,10 @@ Various policies are available but in all environments this is set by default to
 
 * Each day, the system will take a backup, and discard any daily backups older than seven days.
 * Each week, it will relabel the oldest daily backup as a weekly backup and discard any weekly backups older than one
-  month.
+month.
```

Comment:
The formatting was correct before this change.

Reply:
done